vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -171,6 +172,11 @@ def build(self):
                 dash.page_container,
             ],
         )
+        return dmc.MantineProvider(
+            layout,
+            # Use the `theme` to style all Mantine components with a Vizro theme. For more info see https://www.dash-mantine-components.com/components/mantineprovider
+            # theme = {...}
+        )
```

Comment:
```suggestion
return dmc.MantineProvider(
    # Use the `theme` to style all Mantine components with a Vizro theme. For more info see https://www.dash-mantine-components.com/components/mantineprovider theme = {...}
    id="dashboard-container",
    children=[
        html.Div(id="vizro_version", children=vizro.__version__, hidden=True),
        dcc.Store(
            id="vizro_themes",
            data={
                "vizro_dark": pio.templates.merge_templates("vizro_dark", dashboard_overrides),
                "vizro_light": pio.templates.merge_templates("vizro_light", dashboard_overrides),
            },
        ),
        ActionLoop._create_app_callbacks(),
        dash.page_container,
    ],
)
```

Reply:
Would it be OK to wait for the refactor here?  We will likely add more props to the `dmc.MantineProvider` so it might be cleaner to keep the `children` as the variable name `layout` as defined right above the `return` statement.  

 But if you prefer it like your  commit suggestion, I'm ok with that too :-)

Reply:
Ah I see. Then I'm totally okay to keep layout in a separate variable as you originally did.